language/resource.language.uk_ua/strings.po

```diff
@@ -0,0 +1,2256 @@
+msgstr "Налаштування візуалізації"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31214"
+msgid "Series Finale"
+msgstr "Фінал"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31215"
+msgid "Season Finale"
+msgstr "Фінал сезону"
+
+msgctxt "#31216"
+msgid "Video decoder"
+msgstr "Декодер відео"
+
+msgctxt "#31217"
+msgid "Pixel format"
+msgstr "Формат пікселів"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31218"
+msgid "Mid-Season Finale"
+msgstr "Фінал середини сезону"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31219"
+msgid "Left header"
+msgstr "Ліва панель шапки"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31220"
+msgid "Purple"
+msgstr "Фіолетовий"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31221"
+msgid "Slate"
+msgstr "Сланець"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31222"
+msgid "Coal"
+msgstr "Вугільний"
+
+msgctxt "#31223"
+msgid "Gridlines"
+msgstr "Сітка"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31224"
+msgid "Pink"
+msgstr "Рожевий"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31225"
+msgid "Chalk"
+msgstr "Крейдяний"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31226"
+msgid "Blush"
+msgstr "Рум'яний"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31227"
+msgid "Premiere"
+msgstr "Прем'єра"
+
+#: /1080i/Custom_1115_Dialog_ColourPresets.xml
+msgctxt "#31228"
+msgid "Aqua"
+msgstr "Аква"
+
+#: /1080i/Custom_1115_Dialog_ColourPresets.xml
+msgctxt "#31229"
+msgid "Vaporwave"
+msgstr "Вейпорвейв"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31230"
+msgid "Calendar"
+msgstr "Календар"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31231"
+msgid "Customise highlight colours"
+msgstr "Змінити колір підсвітки"
+
+#: /1080i/Custom_1111_Dialog_ColourPicker.xml
+msgctxt "#31232"
+msgid "Gradient"
+msgstr "Градієнт"
+
+msgctxt "#31233"
+msgid "Seek to"
+msgstr "Перейти до"
+
+msgctxt "#31234"
+msgid "Customise indicators"
+msgstr "Редагувати індикатори"
+
+msgctxt "#31235"
+msgid "Indicators"
+msgstr "Індикатори"
+
+msgctxt "#31236"
+msgid "Collection"
+msgstr "Збірка"
+
+msgctxt "#31237"
+msgid "Progress"
+msgstr "Прогрес"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31238"
+msgid "Coral"
+msgstr "Кораловий"
+
+msgctxt "#31239"
+msgid "Footer"
+msgstr "Футер"
+
+#: /1080i/Custom_1115_Dialog_ColourPresets.xml
+msgctxt "#31240"
+msgid "Tropical"
+msgstr "Тропічний"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31241"
+msgid "Colour scheme"
+msgstr "Кольорова гамма"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31242"
+msgid "Blue Slate"
+msgstr "Синій сланець"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31243"
+msgid "Midnight Purple"
+msgstr "Оксамитова ніч"
+
+msgctxt "#31244"
+msgid "Now playing"
+msgstr "Зараз відтворюється"
+
+msgctxt "#31245"
+msgid "Autoscroll labels"
+msgstr "Автопрокрутка міток"
+
+msgctxt "#31246"
+msgid "Autoscroll textboxes"
+msgstr "Автопрокрутка текстових полів"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31247"
+msgid "Colour highlight"
+msgstr "Колір підсвітки"
+
+msgctxt "#31248"
+msgid "Building includes"
+msgstr "Побудова передбачає"
```

Comment:
This one should keep "includes" as is, since it is a technical term (refers to the Includes.xml files).